lib/main.dart

```diff
@@ -12,6 +17,14 @@ void main() async {
   await EasyLocalization.ensureInitialized();
   await AppConfig.loadEnv(Environment.prod);
   await DemoConfig.loadDemoConfig();
+  await Hive.initFlutter();
+  await Hive.initFlutter();
```

Comment:
Why init 2 times?

Reply:
it's duplicated, it'll be removed on the next commit. thanks a @lttung-axonivy